src/actions/wapeSale/getMintlist.ts

```diff
@@ -0,0 +1,67 @@
+import { fetch } from "cross-fetch";
+import { WapeSaleConfig, Maybe, Mintlist } from "../../types";
+import { getWapeSaleContract } from "../../contracts";
+
+const extractIPFSHash = (hay: string) => {
+  const regex = /(Qm.+)$/;
+  const matches = regex.exec(hay);
+  if (!matches) {
+    return undefined;
+  }
+  return matches[1];
```

Comment:
i think 0 is the full match, and 1 is the result of the first group. in this case. .. possibly the same. haha. but since it's using groups, i think that's why the index of 1.